src/dune_util/report_error.ml

```diff
@@ -195,3 +198,8 @@ let report { Exn_with_backtrace.exn; backtrace } =
       | Developer -> append msg (i_must_not_crash ())
     in
     Console.print_user_message msg
+
+let report { Exn_with_backtrace.exn; backtrace } =
+  gen_report exn (Some backtrace)
+
+let report_exception exn = gen_report exn None
```

Comment:
This got me thinking that it might be better to introduce an exception wit reraising like base does:

```ocaml
exception reraise of (exn * Printexc.raw_backtrce)
```

To make exceptions and exceptions with a backtrace interchangeable